src/System.Text.Encodings.Web/tests/ConfigurableScalarTextEncoder.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text.Encodings.Web;
+
+namespace Microsoft.Framework.WebEncoders
```

Comment:
Does this namespace make sense in this context?

Reply:
It's the same namespace as most of the other files in this folder. It's a legacy holdover from when this class was originally owned by the ASP.NET team. We should probably do a sweep of all files in this directory at some point, but I don't think this is the appropriate time.